### ISSUE TITLE
Split de Pagamento

### ISSUE THREAD

**Issue by matheuskildere** [DeveloperCielo/developercielo.github.io#337]
Estou implementando a CIELO E-COMMERCE no marketplace de uma empresa e ao realizar algumas chamadas a API de pagamento completo no cartão de crédito, recebo a seguinte resposta: 
`[
    {
        "Code": 245,
        "Message": "Merchant does not have split configuration"
    }
]`
Obs: Já testei com ambas chaves (MerchantId e MerchantKey), braspag e cielo

Requisição de Teste:

`{
  "MerchantOrderId":"2014111701",
  "Customer":{
      "Name":"Comprador"
   },
  "Payment":{
      "Type":"SplittedCreditCard",
      "Amount":10000,
      "Installments":1,
      "SoftDescriptor":"Marketplace",
      "Capture": true,
      "CreditCard":{
          "CardNumber":"4551870000000181",
          "Holder":"Teste Holder",
          "ExpirationDate":"12/2021",
          "SecurityCode":"123",
          "Brand":"Visa"
      },
      "SplitPayments": [
        {
            "SubordinateMerchantId": "7c7e5e7b-8a5d-41bf-ad91-b346e077f769",
            "Amount": 6000,
            "Fares": {
              "Mdr": 5,
              "Fee": 30
            }
        },
        {
            "SubordinateMerchantId": "2b9f5bea-5504-40a0-8ae7-04c154b06b8b",
            "Amount": 4000,
            "Fares": {
              "Mdr": 4,
              "Fee": 15
            }
        }
    ]
  }
}`

**Comment by souzadavi** [comment]
Matheus o split de pagamento funciona no CIELO E-commerce? Que eu saiba funciona apenas em contas do Braspag.

**Comment by matheuskildere** [comment]
> Matheus o split de pagamento funciona no CIELO E-commerce? Que eu saiba funciona apenas em contas do Braspag.


[DOCUMENTAÇÃO: Split de pagamentos Cielo E-Commerce](https://braspag.github.io//manual/split-de-pagamentos-cielo-e-commerce)

**Comment by souzadavi** [comment]
Sim, na documentação esta informando que o facilitador é a Braspag. Também estou no aguardo dessa funcionalidade dentro da CIELO para implementação no WC Market Place.

![Screenshot from 2020-10-26 11-28-17](https://user-images.githubusercontent.com/636225/97185159-7e775500-177e-11eb-8707-94c92a0f5833.png)

No Split de Pagamentos o responsável pelo fluxo transacional é a Braspag (facilitador).

O **Marketplace se integra à Braspag** para transacionar e informa como será dividida a transação entre cada participante, podendo ser no momento de captura ou em um momento posterior, conhecido como Split pós-transacional, desde que seja dentro de um limite de tempo pré-estabelecido.

Com a transação capturada, a Braspag calcula o valor destinado a cada participante e repassa esses valores, no prazo estabelecido de acordo com cada produto (regime de pagamento*), para cada envolvido na transação.

Se quiser bater um papo to no whats 48 988049344. Abraço.
